core/corehttp/gateway_test.go

```diff
@@ -153,6 +153,7 @@ func TestGatewayGet(t *testing.T) {
 	ns["/ipns/double.example.com"] = path.FromString("/ipns/working.example.com")
 	ns["/ipns/triple.example.com"] = path.FromString("/ipns/double.example.com")
 	ns["/ipns/broken.example.com"] = path.FromString("/ipns/" + k)
+	ns["/ipns/example.man"] = path.FromString("/ipfs/" + k)
```

Comment:
I take it that `.man` has special meaning to the go http library?  I would add a test for this.  To me it is very non-obvious.

Alternatively I would have the content be a small html file and use `example.txt`instead and make sure the content type is an html file not a text file.

Reply:
> I take it that .man has special meaning to the go http library? I would add a test for this. To me it is very non-obvious.
> Alternatively I would have the content be a small html file and use example.txtinstead and make sure the content type is an html file not a text file.

I picked `.man` because it's a valid TLD. I couldn't get this to work with `.html` or `.txt` because IPFS will refuse to resolve these as domains. I considered using `.com` but I needed something that *could* be a COM file (the original issue used a GIF). I didn't want to deal with that.

Reply:
I'll add a comment.